pydy/viz/static/js/dyviz/param_editor.js

```diff
@@ -126,7 +124,7 @@ DynamicsVisualizer.ParamEditor = Object.extend(DynamicsVisualizer, {
 
         jQuery.extend(true,self.model.objects[int_id],updated_object);
         self.Scene.addObjects();
-        self.Scene.addCameras();
+        //self.Scene.addCameras();
```

Comment:
Any reason this is commented out instead of deleted?

Also, do custom cameras that are defined in the Python code work now that this commented out?